embryo/segment_2D_0.py

```diff
@@ -11,8 +11,11 @@
     threshold_local,
     threshold_otsu,
     threshold_multiotsu,
-    try_all_threshold
+    try_all_threshold,
+    sobel
 )
+from skimage.exposure import histogram, equalize_hist
+from skimage import segmentation, morphology
```

Comment:
Actually, update all these imports into `import skimage as ski` to comply with new convention:
https://github.com/scikit-image/scikit-image/blob/main/CONTRIBUTING.rst#stylistic-guidelines